packages/wonder-blocks-tooltip/index.js

```diff
@@ -1,9 +1,12 @@
 // @flow
 import type {Placement} from "./util/types.js";
+import type {PopperElementProps} from "./components/tooltip-bubble.js";
 
 import Tooltip from "./components/tooltip.js";
 import TooltipContent from "./components/tooltip-content.js";
+import TooltipPopper from "./components/tooltip-popper.js";
+import TooltipTail from "./components/tooltip-tail.js";
 
-export {Tooltip as default, TooltipContent};
+export {Tooltip as default, TooltipContent, TooltipPopper, TooltipTail};
```

Comment:
This is really starting to feel like these should be in the same package.

Reply:
About this, I keep thinking about this. Even that Tooltip and Popover share some logic, they serve different purposes. Also, merging both into the same package could lead to mixed API.

e.g.
`import {Tooltip, TooltipContent, Popover, PopoverContent, PopoverContentCore} from "wb-tooltip"`